C/plugins/north/OMF/plugin.cpp

```diff
@@ -877,7 +877,7 @@ uint32_t plugin_send(const PLUGIN_HANDLE handle,
 		Logger::getLogger()->fatal("OMF Endpoint is not available");
 		return 0;
 	}
-Logger::getLogger()->fatal("FIXME: version is %s", version.c_str());
+	// FIXME - The above call is not working. Investigate why? 
```

Comment:
Better to have a Jira for this and add FIXME: FOGL-xxx

Reply:
Created a JIRA 7293.